### PR TITLE
Add scoring.py

### LINKED ISSUE #37: Scoring script

### PR DESCRIPTION
**Pull request by SiqiTao**
Created scoring script which includes:
-  cross-validation scores
-  coefficient table: full, head20(most positive) and tail20(most negative)

Fixed some minor typos/names in model.py

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -99,7 +97,10 @@ def main(input, output):
     save_df(result_df, "score_results", output)
 
     # Storing optimized model
-    pickle.dump(model, open(f"{output}lr_model.rds", "wb"))
+    pickle.dump(model, open(f"{output}final_model.rds", "wb"))
+    
+    # Storing random search
+    pickle.dump(random_search, open(f"{output}random_search.rds", "wb"))
```
Comment (iamMoid) [inline src/model.py:110]: I think we need to revert these changes. The `lr_model.rds` will be the logistic regression model with the best parameters. This should be the output of the `model.py` script. The `feature_selection.py` script imports the `lr_model.rds` performs feature selection and exports the final pipeline as `final_model.rds`, which should then be imported in the `score.py` to score the test data.

File: src/scoring.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# author: DSCI-522 Group-21
```
Comment (iamMoid) [inline src/scoring.py]: Let us rename to `score.py` to tie back to the flowchart

**Review by iamMoid — CHANGES_REQUESTED** [review]

File: src/scoring.py (hunk 2 above)
Comment (SiqiTao) [reply src/scoring.py]: Done

File: src/model.py (hunk 1 above)
Comment (SiqiTao) [reply src/model.py:110]: ok I see.
